packages/react/src/Prose/Prose.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {forwardRef} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+import styles from './Prose.module.css'
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/prose/base.css'
+
+export type ExpectedProseChildren = HTMLDivElement | HTMLHeadingElement | HTMLParagraphElement
+
+export type ProseProps = {
+  /**
+   * Valid children include string encapsulated HTML elements such as `<h1>`, `<p>`, `<div>`, etc.
```

Comment:
Nice. Could we please expand this to include the full list of elements that we automatically style? The only way to figure that out right now is through CSS, so we could perhaps make this easier.

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/316/commits/9a113fa56cfe60cf197437f3f708778c95eb8aab)